Docker/README.md

```diff
@@ -12,22 +12,56 @@ Docker for Mac is the current release of Docker for macOS.
 
 Docker for Mac can be downloaded [here](https://docs.docker.com/docker-for-mac/install/).
 
-## Docker Toolbox
-
-Docker Toolbox is a legacy desktop solution for older Mac and Windows systems that do not meet the requirements of [Docker for Mac](https://docs.docker.com/docker-for-mac/) and Docker for Windows.
-
 ### Prerequisite
 
 You'll need `homebrew-cask` to install Docker Toolbox, if you don't have it refer to [this section](../Homebrew/Cask.md).
 
 ### Installation
 
-    brew cask install docker-toolbox
+These are the steps to install docker using brew
+
+* Install the docker and docker machine from brew
+
+```shell script
+brew install docker docker-machine
```

Comment:
Is `docker-machine` really necessary to mention? I've never used it and for most use-cases it seems like overkill.

Reply:
Using Brew to install docker needs the docker-machine. Reason being: `This is because on macOS the docker binary is only a client and you cannot use it to run the docker daemon, because Docker daemon uses Linux-specific kernel features, therefore you can’t run Docker natively in OS X. So you have to install docker-machine in order to create VM and attach to it. `

more information about this: https://stackoverflow.com/a/49719638

I can add more information about directly using the binary from Dockers website as well. but this option is a leaner method of getting the docker installation.

Reply:
I have added two different ways to install Docker. Please take a look